packages/global-styles/src/components/global-styles-variations/index.tsx

```diff
@@ -113,28 +121,75 @@ const GlobalStylesVariations = ( {
+		translate(
+			'Unlock premium styles and tons of other features with the Premium plan, or try them out now for free.'
+		);
+
+	const premiumStyles = globalStylesVariationsWithoutDefault.map(
+		( globalStylesVariation, index ) => (
+			<GlobalStylesVariation
+				key={ index }
+				showFreeBadge={ false }
+				globalStylesVariation={ globalStylesVariation }
+				isActive={ globalStylesVariation.slug === selectedGlobalStylesVariation?.slug }
+				onSelect={ () => onSelect( globalStylesVariation ) }
+			/>
+		)
+	);
+
+	const headerText = splitPremiumVariations ? translate( 'Default Style' ) : translate( 'Styles' );
+
 	return (
 		<GlobalStylesContext.Provider value={ { base: baseGlobalStyles } }>
-			<div className="global-styles-variations">
-				<GlobalStylesVariation
-					key="base"
-					globalStylesVariation={ baseGlobalStyles }
-					isActive={
-						! selectedGlobalStylesVariation ||
-						isDefaultGlobalStyleVariationSlug( selectedGlobalStylesVariation )
-					}
-					showOnlyHoverView={ showOnlyHoverViewDefaultVariation }
-					onSelect={ () => onSelect( baseGlobalStyles ) }
-				/>
-				{ globalStylesVariationsWithoutDefault.map( ( globalStylesVariation, index ) => (
-					<GlobalStylesVariation
-						key={ index }
-						globalStylesVariation={ globalStylesVariation }
-						premiumBadge={ premiumBadge }
-						isActive={ globalStylesVariation.slug === selectedGlobalStylesVariation?.slug }
-						onSelect={ () => onSelect( globalStylesVariation ) }
-					/>
-				) ) }
+			<div className="global-styles-variations__container">
+				<div
+					className={ classnames( 'global-styles-variations__type', {
+						'combined-variations': ! splitPremiumVariations,
+					} ) }
+				>
+					<div className="global-styles-variations__header">
+						<h2>
+							{ headerText }
+							{ displayFreeLabel && (
+								<div className="global-styles-variations__free-badge">
+									<span>{ translate( 'Free' ) }</span>
+								</div>
+							) }
+						</h2>
+						{ ! splitPremiumVariations && (
+							<div>
+								<p>{ translate( 'You can change your style at any time.' ) }</p>
+							</div>
+						) }
+					</div>
+					<div className="global-styles-variations">
+						<GlobalStylesVariation
+							key="base"
+							showFreeBadge={ displayFreeLabel }
+							globalStylesVariation={ baseGlobalStyles }
+							isActive={
+								! selectedGlobalStylesVariation ||
+								isDefaultGlobalStyleVariationSlug( selectedGlobalStylesVariation )
+							}
+							showOnlyHoverView={ showOnlyHoverViewDefaultVariation }
+							onSelect={ () => onSelect( baseGlobalStyles ) }
+						/>
+						{ ! splitPremiumVariations && premiumStyles }
+					</div>
+				</div>
+				{ splitPremiumVariations && (
+					<div className="global-styles-variations__type">
+						<div className="global-styles-variations__header">
+							<h2> { translate( 'Premium styles' ) } </h2>
+							<div>
+								<p>{ premiumStylesDescription }</p>
+							</div>
```

Comment:
Is the wrapping `div` really needed?
```suggestion
							<p>{ premiumStylesDescription }</p>
```